lib/msal-browser/src/interaction_client/RedirectClient.ts

```diff
@@ -507,6 +507,35 @@ export class RedirectClient extends StandardInteractionClient {
             );
             this.logger.verbose("Auth code client created");
 
+            if(authClient.authority.protocolMode === ProtocolMode.OIDC) {
+                try { 
+                    authClient.authority.endSessionEndpoint;
+                } catch {
+                    if (validLogoutRequest.account?.homeAccountId && validLogoutRequest.postLogoutRedirectUri){
+                        this.browserStorage.removeAccount(validLogoutRequest.account?.homeAccountId);
+                        this.browserStorage.removeAccessToken(validLogoutRequest.account?.homeAccountId);
+                        this.browserStorage.removeIdToken(validLogoutRequest.account?.homeAccountId);
+                        this.browserStorage.removeRefreshToken(validLogoutRequest.account?.homeAccountId);
+                        
+                        this.eventHandler.emitEvent(
+                            EventType.LOGOUT_SUCCESS,
+                            InteractionType.Redirect,
+                            validLogoutRequest
+                        );
+        
+                        if (!this.browserStorage.getInteractionInProgress()) {
+                            this.browserStorage.setInteractionInProgress(true);
+                        }
+                        await this.navigationClient.navigateExternal(
```

Comment:
We don't need to navigate anywhere if there's no logout endpoint. Let's simply log a warning

Reply:
I have this because I want the window to refresh or redirect so the UI doesn't still say the user is logged in. If there's a better way to do that, I'm happy to implement it.

Reply:
Refreshing the UI is the responsibility of the app and doesn't necessarily require a total refresh of the page. I'd rather leave it up to developers to decide how they want to handle that.